test/router-luatest/router_test.lua

```diff
@@ -444,3 +444,72 @@ g.test_ssl = function(g)
     vtest.router_cfg(g.router, cluster_cfg)
     vtest.storage_wait_fullsync(g)
 end
+
+g.test_enable_disable = function(g)
+    --
+    -- gh-291: router enable/disable
+    --
+    local router_name = 'router_1'
+    local router = vtest.router_fiber_new(g, router_name, cluster_cfg,
+        function() ivshard.router.internal.errinj.ERRINJ_CFG_DELAY = true end)
+    -- emulate unconfigured box
+    router:exec(function()
+        old_cfg = box.cfg
+        box.cfg = function(...) return old_cfg(...) end
+    end)
+
+    local status, ret = router:exec(function()
+            static_router = ivshard.router.internal.routers._static_router
+            return pcall(static_router.info, static_router)
```

Comment:
It seems you have indentation step size 8 symbols. Please, set it to 4 (whitespaces, not tabs).